python/hail/keytable.py

```diff
@@ -1076,3 +1077,51 @@ def import_fam(path, quantitative=False, delimiter='\\\\s+', missing='NA'):
         hc = Env.hc()
         jkt = Env.hail().keytable.KeyTable.importFam(hc._jhc, path, quantitative, delimiter, missing)
         return KeyTable(hc, jkt)
+
+    def union(self, *kts):
```

Comment:
No typecheck because you can't refer to KeyTable?  Did you try that trick we thought of yesterday?